app/services/flow/flows/provider_dependants.rb

```diff
@@ -30,10 +30,10 @@ class ProviderDependants < FlowSteps
         has_other_dependants: {
           path: ->(application) { urls.providers_legal_aid_application_means_has_other_dependants_path(application) },
           forward: lambda { |_application, has_other_dependant|
-            has_other_dependant ? :dependants : :own_homes
+            has_other_dependant ? :dependants : :check_income_answers
```

Comment:
Did something change that meant this should no longer be `own_homes`?

Reply:
Yes, so own_homes is the first page of the capital assessment, dependants is the last part of the income section, so when they've finished with dependants they go to the new check income answers page :)